modules/tests/shared/src/main/scala-3/io/circe/tests/CirceMunitSuite.scala

```diff
@@ -0,0 +1,29 @@
+package io.circe.tests
+
+import io.circe.testing.{ ArbitraryInstances, EqInstances }
+import munit.DisciplineSuite
+
+/**
+ * An opinionated stack of traits to improve consistency and reduce boilerplate in circe tests.
+ */
+trait CirceMunitSuite extends DisciplineSuite with ArbitraryInstances with EqInstances with MissingInstances {
```

Comment:
Rather than duplicating `CirceMunitSuite` can you make a `PlatformSpecificInstances` trait for scala-2 and scala-3 and have `CirceMunitSuite` extend it?

Reply:
Ok, I will try to implement this as I could

Reply:
Fixed. I found it's probably not necessary to add an extra trait.